src/App.tsx

```diff
@@ -1,14 +1,31 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { actions as todosActions } from './features/todos';
+import { getTodos } from './api';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { Todo } from './types/Todo';
 import { Loader } from './components/Loader';
+import { TodoModal } from './components/TodoModal';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(true);
+
+  const dispatch = useAppDispatch();
+  const currentTodo: Todo | null = useAppSelector(state => state.currentTodo);
```

Comment:
You don't need to type the `currentTodo', because if you did everything correctly in reducer file, then it knows the type.